changelog.json

```
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "1.6.3": [
+      "[Added] Provide option to discard when files would be overwritten by a checkout - #6755. Thanks @mathieudutour!",
+      "[Fixed] Context menu doesn't open when right clicking on the edges of files in Changes list - #6296. Thanks @JQuinnie!",
```

Avoiding contraction where possible

```suggestion
      "[Fixed] Context menu does not open when right clicking on the edges of files in Changes list - #6296. Thanks @JQuinnie!",
```